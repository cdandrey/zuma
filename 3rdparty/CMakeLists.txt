cmake_minimum_required(VERSION 3.3)

set(INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}")

add_compile_options(${THIRD_PARTY_CXX_FLAGS})

set(SFML_DIR "${CMAKE_CURRENT_SOURCE_DIR}/sfml-2.5.1/lib/cmake/SFML" PARENT_SCOPE)

set(ZUMA_3RDPARTY_INCLUDE_DIRECTORIES "${INCLUDE_DIRS}" PARENT_SCOPE)