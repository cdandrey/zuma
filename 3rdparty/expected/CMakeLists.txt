cmake_minimum_required(VERSION 3.3)

add_library(Expected
    STATIC 
    expected.h
)

target_include_directories(Expected
	PUBLIC
	"${CMAKE_CURRENT_SOURCE_DIR}"
)

target_link_libraries(Expected ZumaLib)